impl/forms/ExtractorDerived.h

```diff
@@ -3206,5 +3206,104 @@ class Extractor<Form_PF_hint> : public ExtractorBase<Form_I>
     uint64_t fixed_field_mask_ = 0;
 };
 
+/**
+ * Atomic Double which has reads a pair of memory locations and can
+ * write a pair of memory locations
+ * RV32 NOTE: rs1 is the address base, rd is the dest data starting register.
+ * The second register added will be rd+1 and rs2+1
+ * Also, checks for alignment of those registers.
+ */
+template<>
+class Extractor<Form_AMO_pair> : public Extractor<Form_AMO>
```

Comment:
Cannot use multiple inheritance of the load & store pair Extractors which require virtual inheritance in the base classes used by many other extractors.  Talked with @klingaard and this is not what folks want to do right now.